api/docs/http_certificate_status.md

```diff
@@ -75,7 +75,7 @@ GET, PUT, DELETE
 
 ### Supported Response Formats
 
-PSON
+`text/pson`
```

Comment:
For this, we could say `application/json`, `text/pson`, and maybe `pson`, since the server should support it too.